docs/guests.md

```diff
@@ -122,9 +122,55 @@ On OpenBSD, the xen drivers are also already part of the kernel. The `install.sh
 For OpenBSD search [the forum](https://xcp-ng.org/forum). See for example [this thread](https://xcp-ng.org/forum/topic/2582/guest-tools-for-openbsd).
 :::
 
-## FreeNAS/TrueNAS
+## TrueNAS 12.0-Release
```

Comment:
I don't think we need two separate sections for FreeNAS and TrueNAS, the steps that I added in #446 should work for all versions after 11 until 12.0-U1. If you want we can do a quick test with an older version.